### ISSUE TITLE
Responsive Controls no longer work in the Popover

### ISSUE THREAD

**Issue by oooh-boi** [elementor/elementor#16312]
Environment:
PHP - 8.0.3
WP - 5.8.1
Elementor - 3.4.3
Elementor PRO - 3.4.1

Additional Custom Breakpoints are ACTIVE under Experiments! 
All available custom breakpoints have been added through the Site Settings.

-----------------------------------

The problem is pretty much straightforward: responsive control inside popover can only work for the Desktop, all other "devices" (custom or the default breakpoints) result in an empty setting/option. Well, it's not empty, it doesn't even get displayed!

-----------------------------------
TEST
A simple test for the plugin developer : 

```
/* ---------- */
$element->add_control(
	'_ob_test',
	[
		'label' => __( 'TEST', 'txtdomain' ),
		'type' => Controls_Manager::POPOVER_TOGGLE,
		'return_value' => 'yes',
	]
);

$element->start_popover();

$element->add_responsive_control(
	'_ob_test_w',
	[
		'label' => __( 'TEST Width', 'txtdomain' ),
		'type' => Controls_Manager::SLIDER,
		'size_units' => [ 'px', '%' ],
		'range' => [
			'px' => [
				'min' => 0,
				'max' => 1000,
			],
			'%' => [
				'min' => 0,
				'max' => 500,
			],
		],
		'default' => [
			'unit' => '%',
			'size' => 50,
		], 
	]
);

$element->end_popover();
/* ---------- */
```

**Comment by shilo-ey** [comment]
Thanks for reporting.

We will address this issue soon and release a patch.

Please follow this thread for updates.

Thanks!

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.4.4

Feel free to update

Thanks!
